src/app/shared/components/keyboard-input/keyboard-input.component.scss

```diff
@@ -1,3 +1,18 @@
+sofie-error-message {
+  display: none;
+}
+
+:host.ng-invalid {
+  &.ng-touched {
+    input {
+      border-color: var(--danger-color);
+    }
+    sofie-error-message {
+      display: block;
+    }
+  }
+}
```

Comment:
Just a suggestion that can be used or not :)
Instead of marking the error message as hidden by default and block when it has been touched, the query can be reversed, such that we want to hide it only when we haven't toucher it. E.g.:
```scss

:host.ng-invalid {
  &.ng-touched {
    input {
      border-color: var(--danger-color);
    }
  }

  &:not(.ng-touched) {
    sofie-error-message {
      display: none;
    }
  }
}
```
It might be that it is not beneficial in some cases.

Reply:
In that specific example we set `display: none;`, only on invalid and not touched inputs, but if the inputs are valid that css is not applied. In similar sematic I can propose that:

```
:host {
  &.ng-invalid {
    &.ng-touched {
      .input {
        border-color: var(--danger-color);
      }
    }
  }

  &.ng-valid,
  &.ng-untouched {
    sofie-error-message {
      display: none;
    }
  }
}
```